### DISCUSSION TITLE
username shows error

### DISCUSSION THREAD

**Discussion by littlestar2125** [JanDeDobbeleer/oh-my-posh#4416] [Troubleshoot] [answered]
Hello,I am new user in use oh-my-posh,when i use themes i found username shows error,
when i change to root,username is wiseli,how can i change this username?
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/48813436/48407dc6-1edc-413a-a6d6-6f7f521018bf)

**Comment by JanDeDobbeleer** [comment, marked as answer]
This is always the `USER` environment variable.

**Reply by littlestar2125** [reply]
if i want to show root@.HotstName,how can i setting the config

**Reply by JanDeDobbeleer** [reply]
There's a `.Root` property you can use in the template.

**Reply by littlestar2125** [reply]
Thanks for your help!
